app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -33,14 +33,15 @@ def can_scale_manually?
       protected
 
       def vmt_candidates
-        vmts = VirtualMachineTemplate.where(
+        vmts = VirtualMachineTemplate.joins(:tenants).where(
           appliance_type: appliance.appliance_type,
-          state: 'active'
+          state: 'active',
+          atmosphere_tenants: { active: true }
         )
         if appliance.tenants.present?
           vmts = restrict_by_user_requirements(vmts)
         end
-        restrict_by_tenant_availability(vmts)
```

Comment:
Actually, `user_selected_tenants` is also used by `restrict_by_user_requirements`. Keeping it.